Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -69,14 +69,37 @@ class VDPortGroup : BaseDSC {
     <#
     .DESCRIPTION
 
+    Specifies the VLAN ID for the Distributed Port Group.
+    Valid values are integers in the range of 1 to 4094.
+    If 0 is specified, the VLAN type is 'None'.
+    #>
+    [DscProperty()]
+    [nullable[int]] $VLanId
+
+    <#
+    .DESCRIPTION
+
     Specifies the name for the reference Distributed Port Group.
     The properties of the new Distributed Port Group will be cloned from the reference Distributed Port Group.
     #>
     [DscProperty()]
     [string] $ReferenceVDPortGroupName
 
+    hidden [string] $RetrieveVDSwitchMessage = "Retrieving distributed switch {0}."
+    hidden [string] $CreateVDPortGroupMessage = "Creating distributed port group {0} on distributed switch {1}."
+    hidden [string] $ModifyVDPortGroupMessage = "Modifying distributed port group {0}."
+    hidden [string] $ModifyVDPortGroupVlanConfigurationMessage = "Modifying the VLAN ID of distributed port group {0} to {1}."
+    hidden [string] $RemoveVDPortGroupMessage = "Removing distributed port group {0} from distributed switch {1}."
+
+    hidden [string] $CouldNotRetrieveVDSwitchMessage = "Could not retrieve distributed switch {0}. For more information: {1}"
+    hidden [string] $CouldNotCreateVDPortGroupMessage = "Could not create distributed port group {0} on distributed switch {1}. For more information: {2}"
+    hidden [string] $CouldNotModifyVDPortGroupMessage = "Could not modify distributed port group {0}. For more information: {1}"
+    hidden [string] $CouldNotModifyVDPortGroupVlanConfigurationMessage = "Could not modify the VLAN ID of distributed port group {0} to {1}. For more information: {2}"
+    hidden [string] $CouldNotRemoveVDPortGroupMessage = "Could not remove distributed port group {0} from distributed switch {1}. For more information: {2}"
+
     [void] Set() {
         try {
+            Write-VerboseLog -Message $this.SetMethodStartMessage -Arguments @($this.DscResourceName)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.